modules/rosa-hcp-create-network.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-creating-cluster-with-aws-kms-key.adoc
+// * rosa_hcp/rosa-hcp-egress-lockdown-install.adoc
+// * rosa_hcp/rosa-hcp-quickstart-guide.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+ifeval::["{context}" == "rosa-hcp-egress-lockdown-install"]
+:egress-lockdown-rosa:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-create-network_{context}"]
+= Creating a Virtual Private Cloud using the ROSA CLI
+
+The `rosa create network` command is available in v.1.2.48 or later of the ROSA command-line interface (CLI). The command uses AWS CloudFormation to create a VPC and the other networking components used to install a ROSA cluster. CloudFormation is a native AWS infrastructure-as-code tool and is compatible with the AWS CLI.
+
+If you do not specify a template, CloudFormation uses a default template that creates the following parameters:
+
+[cols="2a,3a",options="header"]
+|===
+|VPC parameter
+|Value
+
+| Availability zones
+| 1
+
+| Region 
+| `us-east-1`
+  
+| VPC CIDR 
+| `10.0.0.0/16` 
+|===
+
+You can create and customize CloudFormation templates to use with the `rosa create network` command. See the additional resources for information on the default VPC template.
+
+.Prerequisites
+
+* You have configured your AWS account
+* You have configured your Red Hat accounts
+* You have installed the ROSA CLI and configured it to the latest version
+
+.Procedure
+. Create an AWS VPC using the default CloudFormations template by running the following command:
++
+[source,terminal]
+----
+$ rosa create network
+----
+
+. Optional: If you want to modify a parameter, you can specify the parameter using optional argument flags such as `--param Param1=Value1` and `--param Param2=Value2`. For example, if you wanted to modify the region, name, availability zone (AZ) count, and VPC's CIDR parameters, you could run:
++
+[source,terminal]
+----
+$ rosa create network --param Region=us-west-2 --param Name=quickstart-stack --param AvailabilityZoneCount=1 --param VpcCidr=10.0.0.0/16
```

Comment:
Shall we  modify the AvailabilityZoneCount here to 2 or 3 so they have a working example?